libass/ass_fontselect.c

```diff
@@ -625,7 +625,7 @@ static char *select_font(ASS_FontSelector *priv, ASS_Library *library,
                          ASS_FontStream *stream, uint32_t code)
 {
     ASS_FontProvider *default_provider = priv->default_provider;
-    ASS_FontProviderMetaData meta = {0};
+    ASS_FontProviderMetaData meta = { .families = 0 };
```

Comment:
`{0}` is the universal initializer in C, and a compiler shouldn't warn against it. If it does, the warning should be disabled.